packages/realm-web/src/App.ts

```diff
@@ -152,13 +152,13 @@ export class App<
       this._locationUrl = Promise.resolve(this.baseUrl);
     }
     this.localApp = configuration.app;
-    const { storage, transport = new DefaultNetworkTransport() } = configuration;
+    const { storage } = configuration;
```

Comment:
Looks like `storage` is only used once here, could use `configuration.storage` directly on line 166 if you want.